app/build.gradle

```diff
@@ -47,4 +55,7 @@ dependencies {
     testImplementation 'junit:junit:4.12'
     androidTestImplementation "androidx.test:runner:1.1.0-alpha4"
     androidTestImplementation "androidx.test.espresso:espresso-core:3.1.0-alpha4"
+    testImplementation 'org.mockito:mockito-core:2.22.0'
+    androidTestImplementation 'org.mockito:mockito-android:2.22.0'
+    androidTestUtil 'androidx.test:orchestrator:1.1.0-alpha4'
```

Comment:
I think this is released together with androidx.test:runner, therefore it would be worth to create an ext field for the version and use the same for androidx.test:runner.